src/api/Models/Canton.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BDMS.Models;
+
+/// <summary>
+/// Represents a canton in the database.
+/// </summary>
+[Table("cantons")]
+public class Canton
+{
+    /// <summary>
+    /// Gets or sets the <see cref="Canton"/>'s id.
+    /// </summary>
+    [Key]
+    [Column("gid")]
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Canton"/>'s uuid.
+    /// </summary>
+    [Column("uuid")]
+    public string? Uuid { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Canton"/>'s change date.
+    /// </summary>
+    [Column("datum_aend")]
+    public DateOnly? DatumAenderung { get; set; }
```

Comment:
Finds komisch, das hier plötzlich deutsche anstatt englische Spaltennamen verwendet werden bzw. wurden. Aber kann man mal so stehen lassen.

Reply:
Ja, wir können auch jetzt entscheiden die Properties in unserer Applikation zu übersetzen ?

Reply:
Nee, lassen wirs mal so. Vielleicht fällt diese Tabelle eh weg.